mlrun/db/httpdb.py

```diff
@@ -3956,13 +3957,13 @@ def store_alert_config(
         project = project or config.default_project
         endpoint_path = f"projects/{project}/alerts/{alert_name}"
         error_message = f"put alert {project}/alerts/{alert_name}"
-        if isinstance(alert_data, mlrun.common.schemas.AlertConfig):
-            alert_data = alert_data.dict()
+        if isinstance(alert_data, AlertConfig):
+            alert_data = alert_data.to_dict()
         body = _as_json(alert_data)
         response = self.api_call("PUT", endpoint_path, error_message, body=body)
-        return mlrun.common.schemas.AlertConfig(**response.json())
+        return AlertConfig(**response.json())
```

Comment:
For `ModelObj` classes it's better to use `from_dict` here.

Reply:
☝🏼